### PR TITLE
enhancement: store raw data for unknown transaction kinds in order to capture them so they can be parsed later

### PR DESCRIPTION
**Pull request by joel-u410**
This changes the behavior for `unknown` transaction kinds so that they capture the underlying raw data into the `data` column, allowing later analysis & possible parsing in the future without losing necessary data that might get pruned from the node.

Additionally, I've added all existing wasms from the `namada-sdk` to the checksums, so that even if they aren't supported yet in the `transaction` indexer as a known kind, they will at least identify their `name`.

Finally, here I've introduced `on_conflict` when inserting `inner_transactions` and `wrapper_transactions` to make it quite a bit easier to perform some maintenance on an indexer database by simply updating the `last_processed_block` in `crawler_state` and running `transactions` to re-crawl them. (In the future, this could be improved even more by offering a command-line option to re-crawl specific blocks.)

### CODE REVIEW

File: shared/src/checksums.rs [hunk 1]
```diff
@@ -44,6 +49,16 @@ impl Checksums {
             TX_CHANGE_COMMISSION_WASM.to_string(),
             TX_IBC_WASM.to_string(),
             TX_BECOME_VALIDATOR_WASM.to_string(),
+            TX_INIT_ACCOUNT_WASM.to_string(),
+            TX_UNJAIL_VALIDATOR_WASM.to_string(),
+            TX_DEACTIVATE_VALIDATOR_WASM.to_string(),
+            TX_REACTIVATE_VALIDATOR_WASM.to_string(),
+            TX_UPDATE_ACCOUNT_WASM.to_string(),
+            VP_USER_WASM.to_string(),
```
Comment (Fraccaman) [inline shared/src/checksums.rs:57]: i think we can remove this one

**Review by Fraccaman — APPROVED** [review]

File: shared/src/checksums.rs (hunk 1 above)
Comment (joel-u410) [reply shared/src/checksums.rs:57]: Ah sounds good, I can remove -- I had just blindly added all the `wasms` I found in `namada-sdk` but yeah it makes sense that we only need the `tx_` ones.

Comment (joel-u410) [reply shared/src/checksums.rs:57]: Do you know of a way to dynamically query the node for all its `wasms` without knowing their names ahead of time? I was hoping to do that (like what `namadac join-network` does) but it seemed like there wasn't any good way to do it unfortunately.

Comment (joel-u410) [reply shared/src/checksums.rs]: Just updated to remove the `VP_USER_WASM`

Comment (Fraccaman) [reply shared/src/checksums.rs]: good idea! we might be able to do that but lets keep it as a refactor for later :)

**Pull request merged**